libsession/src/main/java/org/session/libsession/messaging/messages/control/DataExtractionNotification.kt

```diff
@@ -0,0 +1,81 @@
+package org.session.libsession.messaging.messages.control
+
+import com.google.protobuf.ByteString
+import org.session.libsignal.libsignal.ecc.ECKeyPair
+import org.session.libsignal.service.internal.push.SignalServiceProtos
+import org.session.libsignal.utilities.logging.Log
+import java.lang.Exception
+
+class DataExtractionNotification(): ControlMessage() {
+    var kind: Kind? = null
+
+    // Kind enum
+    sealed class Kind {
+        class Screenshot() : Kind()
+        class MediaSaved(val timestanp: Long) : Kind()
+
+        val description: String = run {
```

Comment:
Why do you need this `run` keyword here? This should just be a regular computed variable 🤔